index.html

```diff
@@ -0,0 +1,225 @@
+    <script src="https://cdn.jsdelivr.net/npm/bootstrap@5.2.3/dist/js/bootstrap.bundle.min.js"></script>
+    <script src="https://kit.fontawesome.com/936703f17d.js" ></script>
+    <link rel="stylesheet" href="assets/CSS/scss/main.css">
+    <title>Capstone Project</title>
+</head>
+<body>
+    <header class="footbal-menu">
+        <nav class="navbar navbar-expand-md  top-nav">
+        <div class="container-md justify-content-end top-nav-container">
+            <div class="navbar-header">
+                <ul class="navbar-nav">
+                    <li class="nav-item">
+                        <a href="#" class="nav-link text-white"> <i class="fa-brands fa-facebook-f"></i></a>
+                    </li>
+                    <li class="nav-item">
+                        <a href="#" class="nav-link text-white"> <i class="fa-brands fa-twitter"></i></a>
+                    </li>
+                    <li class="nav-item">
+                        <a href="#" class="nav-link text-white"> English </a>
+                    </li>
+                    <li class="nav-item">
+                        <a href="#" class="nav-link text-white"> My Page </a>
+                    </li>
+                    <li class="nav-item">
+                        <a href="#" class="nav-link text-white"> Logout </a>
+                    </li>
+                </ul>
+            </div>
+        </div>
+    </nav>
+    <nav class="navbar main-nav navbar-expand-lg">
+        <div class="container">
+            <a class="navbar-brand d-none d-sm-flex" href="index.html">
+                <img src="assets/images/Logo.png" alt="Afghan Team Logo" width="50px" height="60px">
+            </a>
+            <button class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#collapsibleNavbar">
+                <span class="navbar-toggler-icon"></span>
+            </button>
+            <div class="collapse navbar-collapse justify-content-end collapse-main" id="collapsibleNavbar">
+                <ul class="navbar-nav">
+                    <li class="nav-item">
+                        <a href="#" class="nav-link selected"> Home </a>
+                    </li>
+                    <li class="nav-item">
+                        <a href="#programs" class="nav-link"> Programs </a>
+                    </li>
+                    <li class="nav-item">
+                        <a href="#players" class="nav-link"> Players </a>
+                    </li>
+                    <li class="nav-item">
+                        <a href="#partners" class="nav-link"> Partners </a>
+                    </li>
+                    <li class="nav-item">
+                        <a href="about.html" class="nav-link">About</a>
+                    </li>
+                    <li class="nav-item compaign">
+                        <a href="#" class="nav-link">CC Campaign</a>
+                    </li>
```

Comment:
- [ ] Please align the content of program section elements to match the design


|Your's| Behance design|
|---|---|
|<img width="283" alt="Screenshot 2023-06-28 at 10 32 09 AM" src="https://github.com/Mahdi-Niazi/Capstone-Project-HTML-CSS-JS/assets/44798044/dbbf3041-4d1c-4547-8d91-58e5d81ba17d">|<img width="448" alt="Screenshot 2023-06-28 at 10 40 28 AM" src="https://github.com/Mahdi-Niazi/Capstone-Project-HTML-CSS-JS/assets/44798044/f138a386-b6c2-4d77-8a79-98565ae378a3">|